containers/concent-api/install-dependencies.sh

```diff
@@ -2,22 +2,35 @@
 
 # Update package lists for Debian and install dependencies
 dependencies=(
-    python3-pip
-    python3-virtualenv
-    virtualenv
-
     # Required by psycopg2 python package (PostgreSQL support)
     libpq-dev
 
     # Packages needed to build python extensions, dependencies of golem-messages in particular
     gcc
-    python3-dev
-    libssl-dev
+    libssl1.0.2
+    libssl1.0-dev
     git
+    pkg-config
+    libsecp256k1-dev
+    libffi-dev
+    software-properties-common
+)
+
+# We need to use python3.6. There is no official python3.6 package on the debian stretch,
+# we must include test packages.
+python_dependencies=(
+    python3.6
+    python3.6-dev
+    python3.6-venv
```

Comment:
Put this (including the comment) together with all the other dependencies.

Reply:
I think that when we do this, all libraries will be installed from the test or default repository. The `--target-release` flag can control it, but when we add all dependencies together, we can not control it for a specific library.

Reply:
OK. But in that case you should make this clearer.

Rename the variable to `testing_dependencies`. Put the comment about python 3.6 inside (later we may need to use other packages from testing).